rt/frontend/jaxrs/src/main/java/org/apache/cxf/jaxrs/impl/ResponseImpl.java

```diff
@@ -55,7 +55,6 @@
 import javax.ws.rs.ext.ReaderInterceptor;
 import javax.ws.rs.ext.RuntimeDelegate.HeaderDelegate;
 import javax.xml.stream.XMLStreamReader;
-import javax.xml.transform.Source;
```

Comment:
Should `javax.xml.transform.Source` be part of the `JAXRSUtils::isStreamingOutType` ?